src/ui/atoms/inputBase/InputBase.tsx

```diff
@@ -0,0 +1,61 @@
+// This rule is deactivated because we can not add readonly on the entire ChangeEvent
+/* eslint-disable @typescript-eslint/prefer-readonly-parameter-types */
+import React from 'react'
+import type { RefObject } from 'react'
+import classNames from 'classnames'
+import './inputBase.scss'
+
+export type InputBaseProps = Readonly<{
+  /**
+   * Placeholder text.
+   */
+  readonly placeholder?: string
```

Comment:
Redundant. If you put the definition of that type as argument of `Readonly<>`, then you don't need to declare every field as readonly.